pandas/core/groupby.py

```diff
@@ -1475,9 +1475,88 @@ def describe(self, **kwargs):
     @Substitution(name='groupby')
     @Appender(_doc_template)
     def resample(self, rule, *args, **kwargs):
-        """
-        Provide resampling when using a TimeGrouper
+        """Provide resampling when using a TimeGrouper.
+
         Return a new grouper with our resampler appended
+
+        Parameters
+        ----------
+        rule : str
+            The offset string or object representing target conversion.
+        *args
+            These parameters will be passed to the get_resampler_for_grouping
+            function.
+        **kwargs
+            These parameters will be passed to the get_resampler_for_grouping
+            function.
+
+        Returns
+        -------
+        Grouper
+            Return a new grouper with our resampler appended
+
+        Examples
+        --------
+
+        Start by creating a DataFrame with 9 one minute timestamps.
+        >>> index = pd.date_range('1/1/2000', periods=9, freq='T')
+        >>> df = pd.DataFrame(data=9*[range(4)],
+        ...                   index=index,
+        ...                   columns=['a', 'b', 'c', 'd'])
+        >>> df.iloc[[6], [0]] = 5  # change a value for grouping
+        >>> df
+                             a  b  c  d
+        2000-01-01 00:00:00  0  1  2  3
+        2000-01-01 00:01:00  0  1  2  3
+        2000-01-01 00:02:00  0  1  2  3
+        2000-01-01 00:03:00  0  1  2  3
+        2000-01-01 00:04:00  0  1  2  3
+        2000-01-01 00:05:00  0  1  2  3
+        2000-01-01 00:06:00  5  1  2  3
+        2000-01-01 00:07:00  0  1  2  3
+        2000-01-01 00:08:00  0  1  2  3
+
+        >>> series = pd.Series(range(9), index=index) # delete this
```

Comment:
The convention is to use `s` instead of series, you can check these in the documentation guide. `# delete this` is because you forgot to delete this? it doesn't seem to add value this line.